### PR TITLE
Added activity indicator while image loading in EGOImageView

### PR DESCRIPTION
**Pull request by pomarec**

### CODE REVIEW

**Comment by shnhrrsn** [comment]
This looks pretty good, however, it should be optional.  If EGOImageView is used in a table view, using an activity indicator could be pretty expensive.

**Comment by pomarec** [comment]
You are right. My need is to have it enabled by default. Do you want me to design it as optional or you don't think it worths it ?

File: EGOImageView/EGOImageView.m [hunk 1]
```diff
@@ -44,6 +44,18 @@
...
 - (void)setImageURL:(NSURL *)aURL {
+    // Build activitiIndicator if not existing
+    if (!self.activityIndicator) {
+        activityIndicator = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleWhite];
+        self.activityIndicator.hidesWhenStopped = YES;
+        self.activityIndicator.autoresizingMask = (UIViewAutoresizingFlexibleLeftMargin | UIViewAutoresizingFlexibleRightMargin | UIViewAutoresizingFlexibleTopMargin | UIViewAutoresizingFlexibleBottomMargin);
+        [self addSubview:self.activityIndicator];
+        CGSize activityIndicatorSize = self.activityIndicator.frame.size;
+        self.activityIndicator.frame = CGRectMake(self.frame.size.width/2 - activityIndicatorSize.width/2,
+                                                  self.frame.size.width/2 - activityIndicatorSize.width/2,
```
Comment (cetorres) [inline EGOImageView/EGOImageView.m:55]: Here, should be:
self.frame.size.height/2 - activityIndicatorSize.height/2,

Proportional for height and not for width.

Comment (pomarec) [inline EGOImageView/EGOImageView.m:55]: You are right, thx. Edited on [my fork](https://github.com/pomarec/EGOImageLoading/tree/imageview-activity-indicator).
Anyway this pull request is closed.

**Pull request closed without merging**